index.html

```diff
@@ -29,14 +38,171 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
```

Comment:
-  [ ] Kindly consider removing the card title from the `div`. The `h2` tag is enough to render the title. This will help remove unnecessary code nesting and redundancy.